client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -275,7 +305,13 @@ class ManageMenu extends PureComponent {
 	}
 
 	render() {
-		const menuItems = [ ...this.getDefaultMenuItems(), ...this.getCustomMenuItems() ];
+		const menuItems = orderBy(
+			map(
+				[ ...this.getDefaultMenuItems(), ...this.getCustomMenuItems() ],
+				item => ( ! item.order ? { ...item, order: 0 } : item )
```

Comment:
Why are we reordering?

Reply:
For readability, if we want plugins to be in a certain position I think it's sufficient to leave a comment in the code stating so. It's much easier to read a fixed list, than it is to recalculate order indexes.

Reply:
> Why are we reordering?

`Plugins` should be at the bottom of the Manage section

> It's much easier to read a fixed list, than it is to recalculate order indexes.

The thing is some items are added on the fly, such as `Feedback` and other ones. Anyway, agree we can get the behavior that we want. I thought just adding an order number for each item should be enough to get an ordered list.

Reply:
Isn't this clearer to read then? We probably don't need ordering prop until we have a second use case.
```
[ 
// assuming we remove plugins from this list
...this.getDefaultMenuItems(), 
...this.getCustomMenuItems(). 
getPluginItem(), //plugins is a default item, but should always be last
]
```